src/lib/isRgbColor.js

```diff
@@ -0,0 +1,9 @@
+import assertString from './util/assertString';
+
+const rgbColor = /^rgb\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){2}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\)$/;
+const rgbaColor = /^rgba\((([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]),){3}(0?\.\d|1(\.0)?|0(\.0)?)\)$/;
```

Comment:
Good work on this implementation. 
But, have 1 request, `rgb, percentage based`. 
`E.g rgb(0%,0%,0%)`, I think it is a valid rgb value, which is not covered in this PR. 
Here is a reference material https://www.december.com/html/spec/colorrgbaper.html

Reply:
good point, I'll try to adjust

Reply:
@ezkemboi I have doubts how to tackle the percentage based values. On one hand - it's valid rgb, on the other - e.g. in React Native it's not supported. Would you agree for some kind of optional flag, like `includePercentValues` that would be by default set to true, but will enable to set percentages as invalid value?

Reply:
I totally agree.